src/modules/uavcan/uavcan_params.c

```diff
@@ -41,13 +41,16 @@
 /**
  * Enable UAVCAN.
  *
- * Enables support for UAVCAN-interfaced actuators and sensors.
+ * Allowed values:
+ *  0 - UAVCAN disabled.
+ *  1 - Enabled support for UAVCAN actuators and sensors.
+ *  2 - Enabled support for dynamic node ID allocation and firmware update.
  *
  * @min 0
- * @max 1
+ * @max 2
  * @group UAVCAN
  */
-PARAM_DEFINE_INT32(UAVCAN_ENABLE, 0);
+PARAM_DEFINE_INT32(UAVCAN_ENABLE, 1);
```

Comment:
Because of the induces overall boot delay of 8 seconds the delay should be conditioned on UAVCAN being enabled, and UAVCAN_ENABLED should default to zero. We can do something more aggressive on master using the ORB priority.